macros/apply-tags/unapply_mps_from_tags.sql

```diff
@@ -7,20 +7,25 @@
 
   {% set ns = ns or dbt_tags.get_resource_ns() %}
   {% set tags = dbt_tags.get_adapter_tags(ns=ns) %}
+  {% set column_tags = [] %}
+  {% set policy_data_types_list = var('dbt_tags__policy_data_types', none) -%}
 
-  {% set query -%}
-
-    {% for tag in tags %}
-      {% if loop.first %}
-        create or replace masking policy {{ ns }}.dbt_tags__dummy as (val string) returns string -> val;
-      {% endif %}
+  {% for tag in tags if ".column" in tag["level"] %}
+      {% do column_tags.append(tag) %}
+  {% endfor %}
 
-      alter tag {{ tag }} set masking policy {{ ns }}.dbt_tags__dummy force;
-      alter tag {{ tag }} unset masking policy {{ ns }}.dbt_tags__dummy;
+  {% set query -%}
 
-      {% if loop.last %}
-        drop masking policy {{ ns }}.dbt_tags__dummy;
-      {% endif %}
+    {% for item in column_tags %}
+      {%- if get_masking_policy_for_tag(item.tag) %}
```

Comment:
Can you try to use namespace when calling macros? e.g. dbt_tags. get_masking_policy_for_tag